.github/workflows/plugin-ci.yml

```diff
@@ -96,11 +130,18 @@ jobs:
           path: dist
 
       - name: ci/prepare-artifact
-        run: mv dist/*.tar.gz dist/$GITHUB_REPOSITORY-ci.tar.gz
+        run: mv *.tar.gz ${GITHUB_REPOSITORY#*/}-ci.tar.gz
+        working-directory: dist
 
-      - name: ci/upload-artifact
-        uses: mattermost/actions/artifact-upload@a137067c750390bfc228375dbe745b2aa3a566e8
+      - name: ci/aws-configure
```

Comment:
Since we define this job as `delivery`, should we rename this to `cd/aws-configure` ?

Reply:
that's a good question, it depends based on the discussion we did right? 0/5 but we are going to have a `plugin-cd` for release only.

Reply:
Would prefer to keep things simple and have only CI and CD where CD stands for Deployment. 
But Not strongly opinionated on this. 

We can re-itterate